README.md

```diff
@@ -55,28 +55,55 @@ getUser();
 ```
 **[⬆ back to top](#table-of-contents)**
 
-### Use searchable names
+### Use searchable names (part 1)
+
 We will read more code than we will ever write. It's important that the code we do write is 
 readable and searchable. By *not* naming variables that end up being meaningful for 
 understanding our program, we hurt our readers.
 Make your names searchable.
 
 **Bad:**
+
 ```php
-// What the heck is 86400 for?
-addExpireAt(86400);
+// What the heck is 448 for?
+$result = serialize($data, 448);
```

Comment:
did you mean `json_encode` and `json_decode` below? `serialize` does not have second argument in its signature

http://php.net/manual/ru/function.serialize.php

Reply:
I was referring to the [Symfony Serializer](https://symfony.com/doc/current/components/serializer.html). Now i will correct the example.